metadata/src/main/java/com/redhat/lightblue/metadata/Indexes.java

```diff
@@ -61,6 +63,20 @@ public boolean isEmpty() {
         return indexes.isEmpty();
     }
 
+    public boolean isCaseInsensitiveKey(Path path) {
```

Comment:
This checks if the field is case insensitive in any of the indexes, right? The same field can appear as both case sensitive and case insensitive in multiple indexes, so what is the practical use for this?